practice-one/src/components/introduction.html

```diff
@@ -0,0 +1,18 @@
+<section class="introduction-container">
+
+  <div class="introduction-content-container">
+
+     <div class="people-trust-icon">
+        <img src="./assets/images/trust.png" alt="Heart icon" />
+        <p>People Trust us</p>
+     </div>
+     <div class="header-introduction"><p>We're <span style="color:var(--color-primary);">Serious</span> For <br><span style="color:var(--color-primary);">Food</span> & <span style="color: var(--color-accent);">Delivery</span>.</p></div>
```

Comment:
This hard to read, can use break each element separately line when it come many elements like this. Ex:
```
  <div>
     <p>
     <span>
      <br />
   </div>
```

Reply:
And also do not use inline style

Reply:
Why not use heading tag

Reply:
I changed it , thanks for your contribution